doc/md/getting-started.md

```diff
@@ -611,5 +666,3 @@ Now when we have a graph with data, we can run a few queries on it:
     	return nil
     }
     ```
-
-The full example exists in [GitHub](https://github.com/ent/ent/tree/master/examples/start).
```

Comment:
I prefer it here. Maybe format it as: `#### Full Example`? This will add it to the sidebar:


![image](https://user-images.githubusercontent.com/7413593/125409255-660aa180-e3c4-11eb-8757-0b7fe8d60077.png)

Reply:
Might be better, I'll move it back and add a title.